Userland/Games/GameOfLife/CMakeLists.txt

```
@@ -1,10 +1,12 @@
 compile_gml(GameOfLife.gml GameOfLifeGML.h game_of_life_gml)
 
-set(SOURCES
-    Board.cpp
-    BoardWidget.cpp
-    GameOfLifeGML.h
-    main.cpp
+set(
+  SOURCES
+  main.cpp
+  Board.cpp
+  BoardWidget.cpp
+  GameOfLifeGML.h
+  Pattern.cpp
```

Please keep the previous (and usual) indentation of four spaces.

```suggestion
set(SOURCES
    main.cpp
    Board.cpp
    BoardWidget.cpp
    GameOfLifeGML.h
    Pattern.cpp
```